src/components/GradeLevelFilter.jsx

```diff
@@ -10,40 +10,38 @@ const useStyles = makeStyles((theme) => ({
   },
   popOverWindow: {
     minWidth: '200px',
+    fontSize: '15px',
+    paddingTop: theme.spacing(1),
+    paddingLeft: theme.spacing(1),
+    paddingBottom: theme.spacing(1),
   },
 }));
 
-const gradeList = {
-  12: {
-    value: 'Highschool Senior',
-    isChecked: false,
-  },
-  13: {
-    value: 'College Freshman',
-    isChecked: false,
-  },
-  14: {
-    value: 'College Sophomore',
-    isChecked: false,
-  },
-  15: {
-    value: 'College Junior',
-    isChecked: false,
-  },
-  16: {
-    value: 'College Senior',
-    isChecked: false,
-  },
-};
+const gradeItems = [
+  '12th - High School',
+  'College Freshman',
+  'College Sophomore',
+  'College Junior',
+  'College Senior',
+  '5th Year',
+  'Post Grad',
+];
 
 export default function GradeLevelFilter() {
   const classes = useStyles();
   const [anchorEl, setAnchorEl] = useState(null);
   const handleClick = (event) => setAnchorEl(event.currentTarget);
+  const [selectedGrades, setSelectedGrades] = useState(new Set());
 
-  function updateGradeSelection(value) {
-    gradeList[value].isChecked = !gradeList[value].isChecked;
-    setAnchorEl(false);
+  function updateGradeSelection(gradeSelected) {
+    const newSet = new Set(selectedGrades);
+    if (selectedGrades.has(gradeSelected)) {
+      newSet.delete(gradeSelected);
+      setSelectedGrades(newSet);
+      return;
+    }
+    newSet.add(gradeSelected);
+    setSelectedGrades(newSet);
```

Comment:
```suggestion
    if (selectedGrades.has(gradeSelected)) {
      newSet.delete(gradeSelected);
    } else {
      newSet.add(gradeSelected);
    }
    setSelectedGrades(newSet);
```

I consider the two flows to both be part of the main flow. The single if statement is usually used for error cases or for base cases where most of the work is done in the main flow.

Reply:
Ah ok sounds good I changed it to the` if/else`